Shapes.java

```diff
@@ -1,26 +1,34 @@
-import java.util.*;
-
-public class Shapes{
+import java.lang.Math;
+class AreaCalculator{
   public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
+      squareAreaCalculator(6);
+      System.out.println();
+      circleAreaCalculator(6);
+      System.out.println();
+      triangleAreaCalculator(16,15);
+      System.out.println();
+      heronsFormula(9,11,16);
+      }//end main class
+
+      public static void squareAreaCalculator(double length){
```

Comment:
Can you try and figure out a way to have these methods have a return type that isn't `void`? Something that would make them more modular?